src/adagrams.js

```diff
@@ -1,15 +1,152 @@
+import _ from "lodash";
+
+const letterPool = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoreDict = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10,
+};
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  let tenLetters = [];
+  // let letterPoolCopy = _.cloneDeep(letterPool);
+  let letterPoolCopy = { ...letterPool };
+  while (tenLetters.length !== 10) {
+    // for (let i = 0; i < 10; ++i);
+    // {
+    const keys = Object.keys(letterPool);
+    let randomLetter = keys[Math.floor(Math.random() * keys.length)];
```

Comment:
`keys` grabs all the keys from the object `letterPool` so it's only a list of 26 letters. Instead, we should have a list that represents the distribution of letters. 

`keys` will only have 'E' in it once so when you randomly grab a letter on line 70, you have the same chance of grabbing 'E' as you do 'K'.  When the distribution of letters is accurately represented in a list, there would be 12 Es and 1 K so your chance of randomly getting E would be higher. 

You could build up a list of all letters in a couple of different ways (we saw someone use a spread operator during adagrams review). You could also do something like: 

```javascript
  let letterPoolArray = [];
  for (const [letter, frequency] of Object.entries(letterPool)) {
    for (let i = 0; i < frequency; i++) {
      letterPoolArray.push(letter);
    }
  } 
``` 

after that, then you could randomly get a letter like you do on line 70, but use `letterPoolArray` instead of `keys`

With this approach, you'd need to change your logic on lines 71-73 so that you remove the random letter from `letterPoolArray` so it doesn't get drawn again